src/visitable.cpp

```diff
@@ -164,7 +148,8 @@ VisitResponse visitable<vehicle_cursor>::visit_items( const std::function<VisitR
 }
 
 template <>
-VisitResponse visitable<vehicle_selector>::visit_items( const std::function<VisitResponse( item *, item *, const tripoint* )>& func )
+VisitResponse visitable<vehicle_selector>::visit_items(
```

Comment:
Although such an example is trivially converted to:

```
map_selector sel( pos );
for( map_cursor cur : map_selector( pos ) {
    cur.visit_items( [&cur]( item *it ) {
        g->m.i_at( cur ) ... // where am I?
    } );
}
```

So dropping the third parameter (which I agree is ugly) is certainly an option